doc/SONiC_202006_Release_Notes.md

```diff
@@ -0,0 +1,134 @@
+
+#### FW utils extension: SSD upgrade 
+A modern network switch is a sophisticated equipment which consists of many auxiliary components which are responsible for managing different subsystems (e.g., PSU/FAN/QSFP/EEPROM/THERMAL) and providing necessary interfaces (e.g., I2C/SPI/JTAG).Basically these components are complex programmable logic devices with it's own HW architecture and software. It is very important to always have the latest recommended software version to improve device stability, security and performance. In order to make software update as simple as possible and to provide a nice user frindly interface for various maintenance operations (e.g., install a new FW or query current version) we might need a dedicated FW utility.
+<br> Refer [HLD document](https://github.com/Azure/SONiC/blob/master/doc/fwutil/fwutil.md) and below mentioned PR's for more details. 
+<br> **Pull Requests** : [4764](https://github.com/Azure/sonic-buildimage/pull/4764), [4758](https://github.com/Azure/sonic-buildimage/pull/4758), [941](https://github.com/Azure/sonic-utilities/pull/941), [942](https://github.com/Azure/sonic-utilities/pull/942), [87](https://github.com/Azure/sonic-platform-common/pull/87), [82](https://github.com/Azure/sonic-platform-common/pull/82)
+
+#### Getting docker ready for Debian 10
+
+<br> Refer [HLD document]() and below mentioned PR's for more details. 
+<br> **Pull Requests** : [4671](https://github.com/Azure/sonic-buildimage/pull/4671), [4727](https://github.com/Azure/sonic-buildimage/pull/4727), [4726](https://github.com/Azure/sonic-buildimage/pull/4726), [4665](https://github.com/Azure/sonic-buildimage/pull/4665), [4515](https://github.com/Azure/sonic-buildimage/pull/4515),  [4598](https://github.com/Azure/sonic-buildimage/pull/4598), [4529](https://github.com/Azure/sonic-buildimage/pull/4529), [4480](https://github.com/Azure/sonic-buildimage/pull/4480)
+
+#### Port Mirroring
+This feature describes the high level design details on Port/Port-channel mirroring support, dynamic session management, ACL rules can continue to use port/ERSPAN sessions as the action, Configuration CLI for mirror session.
+<br> Refer [HLD document](https://github.com/Azure/SONiC/blob/e8c86d1b3a03d6320727ff148966081869461e4a/doc/SONiC_Port_Mirroring_HLD.md) and below mentioned PR's for more details. 
+<br> **Pull Requests** : [1314](https://github.com/Azure/sonic-swss/pull/1314), [936](https://github.com/Azure/sonic-utilities/pull/936) 
+
+#### Proxy ARP  
+When an interface is enabled with "proxy_arp", the same is enabled in the kernel. ASIC ARP packet action is also updated to trap these packets to CPU in those interfaces.
+<br> Refer [HLD Document](https://github.com/Azure/SONiC/blob/master/doc/arp/Proxy%20Arp.md) for more details. 
+<br> **Pull Requests** :  [617](https://github.com/Azure/SONiC/pull/617)
+
+#### Pytest 100% moved from ansible to Pytest 
+
+#### SPytest
+
+<br> Refer [HLD Document]() for more details. 
```

Comment:
Doc is here - https://github.com/Azure/sonic-mgmt/blob/master/spytest/Doc/intro.md

Reply:
Done.

Reply:
Ack